webapp/views.py

```diff
@@ -895,9 +923,12 @@ def marketo_submit():
         }
 
     if "acquisition_url" in form_fields:
-        enrichment_fields["acquisition_url"] = form_fields["acquisition_url"]
+        shortened_url = shorten_acquisition_url(form_fields["acquisition_url"])
+        form_fields["acquisition_url"] = shorten_acquisition_url
+        enrichment_fields["acquisition_url"] = shortened_url
     else:
-        enrichment_fields["acquisition_url"] = referrer
+        shortened_url = shorten_acquisition_url(referrer)
+        enrichment_fields["acquisition_url"] = shortened_url
```

Comment:
I don't think the referrer needs to be shortened. Can you compare `referrer` with and without your `short_acquisition_url` see what's the difference?

Reply:
@carkod I checked and concluded that the referrer is the same as the acquisition URL. It's the URL of the page that called the API and contains all URL parameters just like the acquisition URL.